.devcontainer/docker-compose.yml

```diff
@@ -19,10 +19,7 @@ services:
 
     volumes:
       - ..:/workspace:cached
-      - ${TMPDIR}:/tmp/graasp-file-item-storage
-        # On Linux (and other platforms) env var TMPDIR does not exist:
-        # replace "${TMPDIR}" with permanent existing path e.g.
-        # "/tmp/graasp-file-item-storage"
+      - ../tmp:/tmp/graasp-file-item-storage
```

Comment:
Huuum should we revert the changes?

Reply:
I don't think so actually, that was my question: why are we arbitrarily binding the file item storage to some OS-specific local temp storage? I believe it's more convenient to have it bound to a locally visible `tmp` folder which is simply ignored from git and other tools; it also makes debugging nicer since the files can be seen from the editor directly. WDYT? @pyphilia @spaenleh @juancarlosfarah